Plugin/compiler.cpp

```diff
@@ -613,7 +613,7 @@ bool Compiler::IsGnuCompatibleCompiler() const
 void Compiler::AddDefaultGnuComplierOptions()
 {
     // Add GCC / CLANG default compiler options
-    AddCompilerOption("-O", "Optimize generated code. (for speed)");
+    AddCompilerOption("-O", "Optimize generated code (for speed)");

```

Comment:
Since you are touching it, what do you think about something like this?:
```cpp
    AddCompilerOption("-O", "Optimize generated code for speed");
    AddCompilerOption("-O1", "Optimize more for speed");
    AddCompilerOption("-O2", "Optimize even more for speed");
    AddCompilerOption("-O3", "Optimize fully for speed");
    AddCompilerOption("-Os", "Optimize generated code for size");
```

Usually text is put in parentheses when it is not important, and therefore can be safely omitted by the reader. However, in this case it is very important what the optimization target is.